src/main/java/com/aws/greengrass/easysetup/GreengrassSetup.java

```diff
@@ -121,7 +121,9 @@ public class GreengrassSetup {
             + "setup\n"
             + "\t\t\t\t\tsteps and (optional) provisions resources. Defaults to true.\n"
             + "\n\t--trusted-plugin, -tp\t\t(Optional) Path of a plugin jar file. The plugin will be included as "
-            + "trusted plugin in nucleus. Specify multiple times for including multiple plugins.\n";
+            + "trusted plugin in nucleus. Specify multiple times for including multiple plugins.\n"
+            + "\n\t--cert-path\t\t\t(Optional) Path of the certificates that you specify, if you don't specify, "
+            + "then use the default path.\n";
```

Comment:
`(Optional) Path where certificates and keys are written when --provision is true. If no path is specified, the root directory is used.`